### DISCUSSION TITLE
Add an option to replace the Windows 11 command bar in File Explorer with the Windows 10 ribbon

### DISCUSSION THREAD

**Discussion by plofhaan** [valinet/ExplorerPatcher#387] [Ideas]
`reg add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell Extensions\Blocked" /f /v {e2bf9676-5f8f-435c-97eb-11607a5bedf7} /t REG_SZ`

**Comment by valinet** [comment]
Isn't this doing that already?

![image](https://user-images.githubusercontent.com/6503598/141954535-816ca124-40dc-49fe-8f56-4df4c1b28d34.png)

**Comment by plofhaan** [comment]
What I meant to say is, after I added the registry value that I mentioned in my original post, File Explorer ignores the ExplorerPatcher setting marked in your screenshot, in such a way that the ribbon is always shown regardless of whether this setting is set to disabled [×].

To remove this registry value:
`reg delete "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell Extensions\Blocked" /f /v {e2bf9676-5f8f-435c-97eb-11607a5bedf7}`
